### PR TITLE
add marosset to reviewers list

### PR DESCRIPTION
**Pull request by jackfrancis**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

This PR will add @marosset as a CAPZ reviewer.

According to the requirements defined here:

- https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-1

- @marosset has been a member of the Kubernetes org since December 2019:
  - https://github.com/kubernetes/org/pull/1453
- was the primary reviewer of these recent 5 PRs:
  - https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2488
  - https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2270
  - https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2216
  - https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2616
  - https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2193
- in addition was the author of 17 merged PRs:
  - https://github.com/kubernetes-sigs/cluster-api-provider-azure/pulls?q=is%3Apr+author%3Amarosset+is%3Aclosed
- has been a contributing reviewer of 22 PRs overall:
  - https://github.com/kubernetes-sigs/cluster-api-provider-azure/pulls?q=is%3Apr+is%3Aclosed+assignee%3Amarosset+

Mark is a very busy, critical Kubernetes maintainer across the ecosystem, and especially SIG-Windows. The expertise he brings to CAPZ is invaluable, and I would like to formally acknowledge his ongoing efforts by making him a CAPZ reviewer.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -24,3 +24,4 @@ aliases:
     - Jont828
     - jsturtevant
     - sonasingh46
+    - marosset
```
Comment (CecileRobertMichon) [inline OWNERS_ALIASES:27]: nit: list should be alphabetically ordered

**Review by CecileRobertMichon — COMMENTED** [review]
enthusiastic +1!! Thank you @marosset for all your contributions so far

File: OWNERS_ALIASES (hunk 1 above)
Comment (jackfrancis) [reply OWNERS_ALIASES:27]: done!

**Comment by devigned** [comment]
WAT?! This feel like it is a PR from the past. How is this just happening now? +1

/lgtm

**Comment by sonasingh46** [comment]
/lgtm
+1 Thank you @marosset

**Pull request merged**